lib/index.js

```diff
@@ -10,7 +10,7 @@
         date = +date || date;
```

Comment:
Yep I know, that's why the return would be above the date parsing. I suggested to keep your changes.